commit f5f0f7836065c299b1aa1e98d71c9c7a2a5581d3
Author: hexameron <hexameron>
Date:   Tue Mar 20 12:48:10 2018 +0000

    overlays: use all seven dwc2 gadget fifos.
    
    Linux 4.9 needed fifos to be set to their default values,
     leaving the last one (silently) set to zero size.
    From 4.12 Linux allows fifos to be set to any size EXCEPT zero.
    
    Resolves https://github.com/raspberrypi/linux/issues/2390
    
    Signed-off-by: John Greb <h3x4m3r0n@gmail.com>

diff --git a/arch/arm/boot/dts/overlays/dwc2-overlay.dts b/arch/arm/boot/dts/overlays/dwc2-overlay.dts
index 265a16bab008..d6c223bc559a 100644
--- a/arch/arm/boot/dts/overlays/dwc2-overlay.dts
+++ b/arch/arm/boot/dts/overlays/dwc2-overlay.dts
@@ -15,7 +15,7 @@
 			dr_mode = "otg";
 			g-np-tx-fifo-size = <32>;
 			g-rx-fifo-size = <256>;
-			g-tx-fifo-size = <512 512 512 512 512 768>;
+			g-tx-fifo-size = <512 512 512 512 512 256 256>;
 			status = "okay";
 		};
 	};
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1210_f5f0f7836065c299b1aa1e98d71c9c7a2a5581d3.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1210_f5f0f7836065c299b1aa1e98d71c9c7a2a5581d3.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
